# Copied over from the desktop consolidation, which has the comment:
# date:2010-01-27 owner:yippi type:bug state:upstream

--- lynx2-8-7/src/LYMain.c-orig	2010-12-30 16:59:09.933129015 -0600
+++ lynx2-8-7/src/LYMain.c	2010-12-30 16:59:56.506438066 -0600
@@ -57,7 +57,7 @@
 #include <io.h>
 #endif
 
-#if defined(LOCALE) && !defined(HAVE_LIBINTL_H)
+#if defined(LOCALE) && (!defined(HAVE_LIBINTL_H) || !defined(LC_ALL))
 #undef gettext			/* Solaris locale.h prototypes gettext() */
 #include <locale.h>
 #ifndef HAVE_GETTEXT
